### PR TITLE
NEXID-1270 - Updated README.md in all apps

### PR DESCRIPTION
**Pull request by LizanLycan**
**JIRA task url** (if applicable)
https://allianceblockprotocol.atlassian.net/browse/NEXID-1270

**(If Bug) What was the problem being solved?**

**What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Docs

**How to reproduce fix / test this PR?**

### CODE REVIEW

File: kyc_example/README.md [hunk 1]
```diff
@@ -1,58 +1,17 @@
-# NexeraID Identity-SDK
+# KYC Example App
 
-This repo shows how to use a NexeraID Identity-SDK to integrate a Identity flow in your application. 
-This example uses Nextjs and React.
+This is the most basic KYC example flow that uses the SDK of NexeraID
```
Comment (AndrewMcFarlandCampbellNexeraID) [inline kyc_example/README.md:3]:
```suggestion
This is a simple KYC example flow, using the NexeraID Identity SDK.
```

File: defi_example/README.md [hunk 2]
```diff
@@ -1,30 +1,17 @@
-# Create T3 App
+# DEFi Example App
 
-# For DEFI App
+This is an example defi app that requires signer interaction with web3 modal and has swapping example process
```
Comment (AndrewMcFarlandCampbellNexeraID) [inline defi_example/README.md:3]:
```suggestion
This is an example DeFi app that requires signer interaction with a Web3 modal, and has an example swapping process.
```

File: bank_web3_example/README.md [hunk 3]
```diff
@@ -1,30 +1,17 @@
-# Create T3 App
+# Bank Web3 Example App
 
-# WEB3 Bank App
+This is an example bank web3 app that requires signer interaction with web3 modal
```
Comment (AndrewMcFarlandCampbellNexeraID) [inline bank_web3_example/README.md:3]:
```suggestion
This is an example Web3 bank app that requires signer interaction with a Web3 modal.
```

File: bank_example/README.md [hunk 4]
```diff
@@ -1,30 +1,17 @@
-# Create T3 App
+# Bank Example App
 
-This is a [T3 Stack](https://create.t3.gg/) project bootstrapped with `create-t3-app`.
+This is an example bank app that does not require signer interaction with web3 modal
```
Comment (AndrewMcFarlandCampbellNexeraID) [inline bank_example/README.md:3]:
```suggestion
This is an example bank app that does not require signer interaction with a Web3 modal.
```

**Comment by joelamouche** [comment]
Pls also add links to the readmes in the main readme (or copy them)

File: bank_web3_example/README.md [hunk 5]
```diff
@@ -1,30 +1,17 @@
-# Create T3 App
+# Bank Web3 Example App
```
Comment (AndrewMcFarlandCampbellNexeraID) [inline bank_web3_example/README.md:1]:
```suggestion
# Web3  Bank Example App
```

This sounds slightly better to me

**Review by AndrewMcFarlandCampbellNexeraID — APPROVED** [review]

**Pull request merged**